personality/argus/pap.pan

```diff
@@ -44,105 +44,108 @@ variable PAP_LOCATION_SBIN = PAP_LOCATION + '/sbin';
 
 variable PAP_CONFIG_FILE = PAP_HOME + '/conf/pap_configuration.ini';
 variable PAP_CONFIG_CONTENTS = {
-  contents = "#\n";
-  contents = contents + '# PAP configuration' + "\n";
-  contents = contents + '# ' + "\n";
-  contents = contents + '# Documentation: https://twiki.cern.ch/twiki/bin/view/EGEE/AuthZPAPConfig' + "\n";
-  contents = contents + '# ' + "\n";
-  contents = contents + '[paps]' + "\n";
-  contents = contents + '## Trusted PAPs will be listed here' + "\n";
-  ok = first(PAPS,name,properties);
-  while (ok) {
-    contents = contents + name + '.type = '    + properties['type']               + "\n";
-    contents = contents + name + '.enabled = ' + to_string(properties['enabled']) + "\n";
-    contents = contents + name + '.dn = '      + properties['dn']                 + "\n";
-    contents = contents + name + '.hostname = '+ properties['hostname']           + "\n";
-    contents = contents + name + '.port = '    + to_string(properties['port'])    + "\n";
-    contents = contents + name + '.path = '    + properties['path']               + "\n";
-    contents = contents + name + '.protocol = '+ properties['protocol']           + "\n";
-    contents = contents + name + '.public = '  + to_string(properties['public'])  + "\n";
-    ok = next(PAPS,name,properties);
-  };
-  contents = contents + "\n";
-  contents = contents + '[paps:properties]' + "\n";
-  contents = contents + 'poll_interval = ' + to_string(PAP_POLL_INTERVAL) + "\n";
-  contents = contents + 'ordering = ' + PAP_ORDERING + "\n";
-  contents = contents + "\n";
-  contents = contents + '[repository]' + "\n";
-  contents = contents + 'location = ' + PAP_REPO_LOCATION + "\n";
-  contents = contents + 'consistency_check = ' + PAP_CONSISTENCY_CHECK + "\n";
-  contents = contents + 'consistency_check.repair = ' + PAP_CONSISTENCY_CHECK_REPAIR + "\n";
-  contents = contents + "\n";
-  contents = contents + '[standalone-service]' + "\n";
-  contents = contents + 'entity_id = ' + PAP_ENTITY_ID + "\n";
-  contents = contents + 'hostname = ' + PAP_HOST + "\n";
-  contents = contents + 'port = ' + to_string(PAP_PORT) + "\n";
-  contents = contents + 'shutdown_port = ' + to_string(PAP_SHUTDOWN_PORT) + "\n";
-  if (is_defined(PAP_SHUTDOWN_COMMAND)) {
-   contents = contents + 'shutdown_command = ' + PAP_SHUTDOWN_COMMAND + "\n";
-  };
-  contents = contents + "\n";
-  contents = contents + '[security]' + "\n";
-  contents = contents + 'certificate = ' + SITE_DEF_HOST_CERT + "\n";
-  contents = contents + 'private_key = ' + SITE_DEF_HOST_KEY + "\n";
-
-  contents;
+    contents = "#\n";
```

Comment:
This is a standard INI file, metaconfig with `tiny` would be much cleaner for this.
Which is exactly what happens below... I'm confused.

Reply:
What happens below has been added in 2014 and probably for test (see commit c2d5b3d39cf4239b29cfe97f17cb66f5bb958172). The generated file is not correct, as one of the section name is 'paps:properties'. It can be used as dict entry only if it is correctly escaped, but does not get unescape later by the metaconfig component, leading to such result: 
```
[paps_3aproperties]
ordering=ngi, default
pool_interval=14400
```

Reply:
Ah, I see. @stdweird worth an issue on this for metaconfig?